src/ralph/security/transitions.py

```diff
@@ -16,9 +15,4 @@
 
 @transition_action(verbose_name=_("Cleanup security scans"), models=HOST_MODELS)
 def cleanup_security_scans(cls, instances, **kwargs):
-    with transaction.atomic():
-        for instance in instances:
-            try:
-                instance.securityscan.delete()
-            except cls.securityscan.RelatedObjectDoesNotExist:
-                pass
+    pass
```

Comment:
is this intended?

Reply:
Yes - we stopped using security scans and this didn't work. We will remove the securityscan data soon.